request_a_govuk_domain/request/templates/confirm.html

```diff
@@ -33,15 +33,132 @@ <h2 class="govuk-heading-m">Registrar details</h2>
+              {% if registration_data.registrant_type == 'joint_authority' %}Joint Authority{% endif %}
+              {% if registration_data.registrant_type == 'joint_committee' %}Joint Committee{% endif %}
+              {% if registration_data.registrant_type == 'representative' %}Representing public sector bodies{% endif %}
+            {% endgds_summary_list_row_value %}
+            {% gds_summary_list_row_actions %}
+              {% gds_summary_list_row_actions_item_inline href="/" %}
+                <a href="/registrant-type?change">Change</a>
+              {% endgds_summary_list_row_actions_item %}
+            {% endgds_summary_list_row_actions %}
+          {% endgds_summary_list_row %}
+
+          {###### Organisation name ######}
+
+          {% gds_summary_list_row %}
+            {% gds_summary_list_row_key %}Organisation name{% endgds_summary_list_row_key %}
+            {% gds_summary_list_row_value %}{{ registration_data.registrant_organisation_name }}{% endgds_summary_list_row_value %}
+            {% gds_summary_list_row_actions %}
+              {% gds_summary_list_row_actions_item_inline href="/" %}
+                <a href="/change-registrant">Change</a>
+              {% endgds_summary_list_row_actions_item %}
+            {% endgds_summary_list_row_actions %}
+          {% endgds_summary_list_row %}
+
+
+          {###### Reason for request ######}
+
+          {% if reason_for_request %}
+            {% gds_summary_list_row %}
+              {% gds_summary_list_row_key %}Reason for request{% endgds_summary_list_row_key %}
+              {% gds_summary_list_row_value %}
+                {% if reason_for_request == "website-email" %}Website and email address{% endif %}
+                {% if reason_for_request == "email-only" %}Email only{% endif %}
+              {% endgds_summary_list_row_value %}
+              {% gds_summary_list_row_actions %}
+                {% gds_summary_list_row_actions_item_inline href="/" %}
+                  <a href="/domain-purpose?change">Change</a>
+                {% endgds_summary_list_row_actions_item %}
+              {% endgds_summary_list_row_actions %}
+            {% endgds_summary_list_row %}
+          {% endif %}
+
+
+          {###### Exemption ######}
+
+          {% if central_gov %}
+            {% gds_summary_list_row %}
+              {% gds_summary_list_row_key %}Exemption from GOV.UK website{% endgds_summary_list_row_key %}
+              {% gds_summary_list_row_value %}
+                <a href="/media/{{ registration_data.exemption_file_uploaded_filename }}" target="_blank">{{ registration_data.exemption_file_original_filename }}</a>
+              {% endgds_summary_list_row_value %}
+              {% gds_summary_list_row_actions %}
+                {% gds_summary_list_row_actions_item_inline href="/" %}
+                  <a href="/exemption?change">Change</a>
```

Comment:
Some of the similar places don't have ? in them...e,g, change-written-permission

Reply:
yes. There are a couple of `?change` remaining, where it was unclear what the behaviour was with "back to answers". I think that it's obsolete now with v2, so I'm going to merge this and I reckon most of those links will go anyway.